src/dex-ui/store/swapSlice/swapSlice.ts

```diff
@@ -233,7 +236,8 @@ const createSwapSlice: SwapSlice = (set, get): SwapStore => {
       if (swap.pairInfo.precision === undefined) {
         throw Error("Precision not found");
       }
-      const preciseSlippage = valueToPercentAsNumberWithPrecision(slippageTolerance, swap.pairInfo.precision);
+      // const preciseSlippage = valueToPercentAsNumberWithPrecision(slippageTolerance, swap.pairInfo.precision);
+      const preciseSlippage = BigNumber(slippageTolerance).times(swap.pairInfo.precision);
```

Comment:
Need a discussion on this method `valueToPercentAsNumberWithPrecision` from my understanding the user input `slippage` is a number and the `slippage` we get is a number so while sending `slippageTolerance` we need to multiply it correct precision with `valueToPercentAsNumberWithPrecision ` method the actual `slippageTolerance ` given by user is shifted by 2 extra which fails the transaction for the user with slippage Tolerance error.